rest-api/dao/metrics_cache_dao.py

```diff
@@ -242,15 +243,16 @@ def to_client_json(self, result_set):
   def get_metrics_cache_sql(self):
     sql = """insert into metrics_gender_cache """
     gender_names = ['UNSET', 'Woman', 'Man', 'Transgender', 'PMI_Skip', 'Non-Binary',
-                    'Other/Additional Options']
+                    'Other/Additional Options', 'Prefer not to say']
     gender_conditions = [
-      ' (ps.gender_identity_id IS NULL OR ps.gender_identity_id=924) ',
```

Comment:
I'm not sure that we can expect id's to never change.

Reply:
done